src/BingAIClient.js

```diff
@@ -180,7 +180,7 @@ export default class BingAIClient {
             onProgress = () => { };
         }
 
-        if (jailbreakConversationId || !conversationSignature || !conversationId || !clientId) {
```

Comment:
This is not how the logic is intended to work. `jailbreakConversationId` creates a new conversation on purpose every message and restores the message history by sending the locally stored messages as context to Bing.